src/routes/InventoryTable.js

```diff
@@ -48,7 +47,7 @@ const Inventory = (props) => {
       <PageHeader className="pf-m-light">
         <PageHeaderTitle title="Systems" />
       </PageHeader>
-      <Main>
+      <section className="pf-v5-l-page__main-section pf-v5-c-page__main-section">
```

Comment:
Should/Could we maybe use the `PageSection` component here?

https://www.patternfly.org/components/page#pagesection

Reply:
yup, replaced it on more places